src/views/homepage/components/certifications-section/components/split-layout-with-background/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode } from 'react'
+import classNames from 'classnames'
+import s from './split-layout-with-background.module.css'
+
+/**
+ * Render a split layout with a background.
+ */
+export function SplitLayoutWithBackground({
+	paddingClass,
+	backgroundClass,
+	startSlot,
+	endSlot,
+}: {
```

Comment:
Nit: would be easier to read if this was defined as its own interface.

Reply:
Good call - split out to a separate `types.ts` file 👍